app/src/organisms/ModuleWizardFlows/SelectLocation.tsx

```diff
@@ -116,32 +130,36 @@ export function SelectLocation(props: SelectLocationProps): JSX.Element {
       moduleFixtures
     )
     if (!isEqual(selectedFixtureIdByCutoutIds, configuredFixtureIdByCutoutId)) {
-      updateDeckConfiguration(
-        deckConfig.map(cc => {
-          if (cc.cutoutId in configuredFixtureIdByCutoutId) {
-            let replacementFixtureId: CutoutFixtureId = SINGLE_LEFT_SLOT_FIXTURE
-            if (SINGLE_CENTER_CUTOUTS.includes(cc.cutoutId)) {
-              replacementFixtureId = SINGLE_CENTER_SLOT_FIXTURE
-            } else if (SINGLE_RIGHT_CUTOUTS.includes(cc.cutoutId)) {
-              replacementFixtureId = SINGLE_RIGHT_SLOT_FIXTURE
-            }
-            return {
-              ...cc,
-              cutoutFixtureId: replacementFixtureId,
-              opentronsModuleSerialNumber: undefined,
-            }
-          } else if (cc.cutoutId in selectedFixtureIdByCutoutIds) {
-            return {
-              ...cc,
-              cutoutFixtureId:
-                selectedFixtureIdByCutoutIds[cc.cutoutId] ?? cc.cutoutFixtureId,
-              opentronsModuleSerialNumber: attachedModule.serialNumber,
-            }
-          } else {
-            return cc
+      const updatedDeckConfig = deckConfig.map(cc => {
+        if (cc.cutoutId in configuredFixtureIdByCutoutId) {
+          let replacementFixtureId: CutoutFixtureId = SINGLE_LEFT_SLOT_FIXTURE
```

Comment:
```suggestion
          const replacementFixtureId: CutoutFixtureId = SINGLE_LEFT_SLOT_FIXTURE
```
No let bindings please. Let's spread in the whole object in each case.